README.md

```diff
@@ -98,6 +98,8 @@ The ws://cable.example.com address must point to your set of Action Cable server
 must share a cookie namespace with the rest of the application (which may live under http://example.com).
 This ensures that the signed cookie will be correctly sent.
 
+If the Action Cable server is running on the rails application, a relative url can be used. It will use the approperiate protocol.
```

Comment:
Sorry for the runaround, but I think it would be better to remove this and update the `createConsumer` example [here in the README](https://github.com/rails/actioncable/blob/master/README.md#in-app) to `App.cable.createConsumer("/websocket")` instead of piecing together a URL. No need to explain more than that.